src/server/socketapisocket_mac.mm

```diff
@@ -174,28 +177,36 @@ - (void)connectToLoginAgent {
     // Setup our connection to the launch item's service
     // This will start the launch item if it isn't already running
     NSLog(@"[KD] Setup connection with login item agent");
-    NSBundle *appBundle = [NSBundle bundleForClass:[self class]];
-    NSString *loginItemAgentMachName = [appBundle objectForInfoDictionaryKey:@"LoginItemAgentMachName"];
-    if (!loginItemAgentMachName) {
-        NSLog(@"[KD] LoginItemAgentMachName undefined");
-        return;
-    }
-
-    NSError *error = nil;
-    _loginItemAgentConnection = [[NSXPCConnection alloc] initWithLoginItemName:loginItemAgentMachName error:&error];
-    if (_loginItemAgentConnection == nil) {
-        NSLog(@"[KD] Failed to connect to login item agent: %@", [error description]);
-        return;
-    }
-
-    /*
-    // To debug with an existing login item agent
-    _loginItemAgentConnection = [[NSXPCConnection alloc] initWithMachServiceName:loginItemAgentMachName options:0];
-    if (_loginItemAgentConnection == nil) {
-        NSLog(@"[KD] Failed to connect to login item agent");
-        return;
+    NSString *loginItemAgentMachName;
```

Comment:
Perhaps initialize this value to `nil` and check its validity after the if statement

Reply:
Done